tests/api_tests/test_base.py

```diff
@@ -1,10 +1,113 @@
 # -*- coding: utf-8 -*-
 from nose.tools import *  # flake8: noqa
-from tests.base import OsfTestCase
 
+from website.models import Node
+from tests.base import OsfTestCase
+from tests.factories import UserFactory, ProjectFactory, FolderFactory, DashboardFactory
 
 class TestApiBaseViews(OsfTestCase):
 
     def test_root_returns_200(self):
         res = self.app.get('/api/v2/')
         assert_equal(res.status_code, 200)
+
+
+class TestFiltering(OsfTestCase):
```

Comment:
Indeed. These are integration tests (routes + views + filter backend + response format). I agree that more isolated unit tests for the FilterBackend would be valuable.